Makefile

```diff
@@ -126,3 +127,8 @@ deploy/provision:
 deploy/app:
 	@echo "Deploying version $(V) to a remote server"
 	$(DEPLOY_CMD) playbooks/03_app.yml --extra-vars "app_version=$(V)"
+
+deploy/lint:
```

Comment:
Why? Liniting should happen before the files get deployed?
See: https://blog.42mate.com/installing-and-using-pre-commit-with-ansible-lint/

Reply:
Or before they get pushed, yes. The `deploy/` here is to separate from `make lint` which lints the source/app. 
Another option would be to make it `lint/deploy` target or something like that. WDYT?

Reply:
Aehm yes. Before push. Or at least before merge.
So either it is just "lint" if you want to execute manually. But I will forget to do this. Or you add a pre-commit hook to it. Like at website.

Reply:
Yes, and the next step would be to add it here: https://github.com/EuroPython/internal-bot/pull/3/files :)

It currently only runs tests, but the goal is to also run linter as part of the same checks.

Reply:
Fixed after rebasing on the new CI/CD setup. Now lint/deploy is run as part of build